docs/faq.rst

```diff
@@ -0,0 +1,21 @@
+Frequently-Asked Questions
+==========================
+
+When I try to run my Python file from the shell or Brickman,
+it throws errors or exits immediately
+-----
```

Comment:
You can tell I was distracted while writing this :laughing: Three times I wrapped this line, and three times I realized it was broken and fixed it.